mps_youtube/player.py

```diff
@@ -138,7 +138,7 @@ def _launch_player(self):
         pass
 
     def send_metadata_mpris(self):
-        metadata = util._get_metadata(self.song.title)
+        metadata = util._get_metadata(self.song.title) if config.LOOKUP_METADATA.get == True else None
```

Comment:
It would be more idiomatic to leave out the ` == True` when testing a boolean condition.